src/typescript-transpile.js

```diff
@@ -0,0 +1,152 @@
+// From https://github.com/teppeis/typescript-simple/blob/master/index.js
```

Comment:
After doing so, you can run `script/grunt lint` from the root of the repository to ensure there are no linter errors.